test/Microsoft.AspNet.Mvc.Core.Test/Formatters/XmlDataContractSerializerInputFormatterTest.cs

```diff
@@ -359,6 +366,118 @@ public void XmlDataContractSerializerFormatterThrowsWhenMaxDepthIsBelowOne()
             Assert.Equal("Test Error 3", serializableError["key2"]);
         }
 
+        [Fact]
+        public async Task XmlDataContractSerializerFormatterThrowsWhenNotConfiguredWithRootName()
```

Comment:
You can remove the prefix `XmlDataContractSerializerFormatter` as its redundant(as the file name is grouping the tests) and makes it difficult to read